### PR TITLE
adding  batch_norm to pytorch frontend 

### PR DESCRIPTION
**Pull request by VikramxD**

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Review by CheezBallzPi — CHANGES_REQUESTED** [review]
Your formatting test and frontend tests are not running. It seems you have a problem in your decorator statement. Have a look at the test logs.

**Comment by CheezBallzPi** [comment]
Closing because stale.
